core/src/main/java/org/axonframework/queryhandling/MonoWrapper.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.queryhandling;
+
+import reactor.core.publisher.Mono;
+
+import java.util.function.Consumer;
+
+/**
+ * Wrapper around {@link Mono}. Since project-reactor is not required dependency in this Axon version, we need wrappers
```

Comment:
Typo: `Since project-reactor is not a required...`. In short, I think you should add `a`.